Same patch as for systemd in commit
7144f2f04b705538a893e538a6b851f536f433b6:

Fix deactivation of gtk-doc

The tarball contains the Makefile for building documentation with gtk-doc,
Unfortunately the AM_CONDITIONAL variable is not the correct one, which
results in an error when running autoreconf.

This patch fixes this issue.

Signed-off-by: Eric Le Bihan <eric.le.bihan.dev@free.fr>
Signed-off-by: Hadrien Boutteville <hadrien.boutteville@gmail.com>

--- a/gtk-doc.make
+++ b/gtk-doc.make
@@ -267,7 +267,7 @@
 #
 # Require gtk-doc when making dist
 #
-if HAVE_GTK_DOC
+if ENABLE_GTK_DOC
 dist-check-gtkdoc: docs
 else
 dist-check-gtkdoc:
